src/test/java/de/filefighter/rest/cucumber/ViewFolderContentsSteps.java

```diff
@@ -60,4 +60,42 @@ public void theResponseContainsAnEmptyListForFilesAndFolders() throws JsonProces
 
         assertTrue(rootNode.isEmpty());
     }
+
+    @And("the response not contains the file with fileSystemId {long} and name {string}")
+    public void theResponseNotContainsTheFileWithFileSystemIdAndName(long fsItemId, String name) throws JsonProcessingException {
+
+        ArrayNode rootNode = (ArrayNode) objectMapper.readTree(latestResponse.getBody());
+        if (!rootNode.isContainerNode())
+            throw new AssertionError("Response was not an Array or empty.");
+
+        if
+        (rootNode.isEmpty()) {
+            assertTrue(true);
+        } else {
+            boolean notFound = true;
+            for (JsonNode node : rootNode) {
+                if (node.get("fileSystemId").asLong() == fsItemId &&
+                        node.get("name").asText().equals(name) &&
+                        node.get("type").asText().equals("FOLDER")) // WTF why check for folder
```

Comment:
don't forget

Reply:
ah right i only changed it in one place.